db/seeds.rb

```diff
@@ -3,6 +3,92 @@
 
 exit if Rails.env.test?
 FactoryBot.create :user, first_name: 'David', last_name: 'Faulkenberry', staff: true, spire: '12345678@umass.edu', email: 'dfaulken@umass.edu', admin: true
+user = FactoryBot.create :user, first_name: 'Justin', last_name: 'Forgue', spire: '13579246@umass.edu', email: 'jforgue@umass.edu'
+department = FactoryBot.create :department, name: 'Bus'
+position = FactoryBot.create :position, department: department, name: 'Operator'
+FactoryBot.create :application_submission, data: [["Bus Application", nil, "heading", 1],
```

Comment:
I guess this is great but I _hate_ that the actual bus application is the ApplicationTemplate we have on seed. Filling it out for development is a stupid chore because of how many questions there are.

Reply:
There's no programmatic reason to have all the various questions about licensing or the information about drug testing--the only thing that's necessary is to have an example of all the different question types (date selecting, date-time, text-area, etc).

Reply:
> some sort of mock application that's not so long

YES. Where the questions aren't all required.